botorch/optim/initializers.py

```diff
@@ -670,7 +670,7 @@ def gen_one_shot_hvkg_initial_conditions(
     # sampling from the optimizers
     eta = options.get("eta", 2.0)
     if num_optim_restarts > 0:
-        probs = torch.nn.functional.softmax(eta * standardize(fantasy_vals))
+        probs = torch.nn.functional.softmax(eta * standardize(fantasy_vals), dim=-1)
```

Comment:
`-1` is just the default behavior when you call this on a multi-dimensional tensor, so I went with that. The alternative is `0` but we do need to pass something to avoid the warning.

Reply:
The default is `dim=0` for tensors that are 0d, 1d, or 3d and otherwise 1, for some reason. https://github.com/pytorch/pytorch/blob/7d5e8c1d5131d3dfbcd47bb21748563346929b03/torch/nn/functional.py#L1790

Reply:
Weird. Let me use 0 then to be consistent with that (whatever that means).